dashboard/src/components/IssueDetails/IssueDetails.tsx

```diff
@@ -14,20 +16,32 @@ import { getMiscSection } from '@/components/Section/MiscSection';
 
 import { useIssueDetails } from '@/api/issueDetails';
 
+import type { TableFilter, TestsTableFilter } from '@/types/tree/TreeDetails';
+
+import { IssueDetailsTestSection } from './IssueDetailsTestSection';
+
 interface IIssueDetails {
   issueId?: string;
   versionNumber?: string;
+  tableFilter: TableFilter;
+  onClickTestFilter: (filter: TestsTableFilter) => void;
+  getTestTableRowLink: (testId: string) => LinkProps;
 }
 
 export const IssueDetails = ({
   issueId,
   versionNumber,
+  tableFilter,
+  onClickTestFilter,
+  getTestTableRowLink,
 }: IIssueDetails): JSX.Element => {
   const { data, error, isLoading } = useIssueDetails(
     issueId ?? '',
     versionNumber ?? '',
   );
 
+  const hasTest = data?.test_status !== null;
```

Comment:
Wouldn't `data?.test_status` return undefined when it's empty?

Reply:
you're right, I'll change the comparison to consider the test_status only when there is any data